env/common/templates/galaxy/files/xsede_report.py.j2

```diff
@@ -0,0 +1,179 @@
+#!/usr/bin/env  python
+"""
+!! This file is templated by Ansible, manual changes will have no effect.
+
+Report jobs to XSEDE API. Run with CRON and in Galaxy venv.
+"""
+from __future__ import print_function
+
+import argparse
+import os
+import sys
+
+import psycopg2
+import requests
+
+galaxy_root = os.path.abspath(os.path.join(os.path.dirname(__file__), os.pardir))
+sys.path.insert(1, os.path.join(galaxy_root, 'lib'))
```

Comment:
This won't be correct, right? If this is in `/srv/galaxy/<instance>/bin` then Galaxy is in `/cvmfs/<instance>.galaxyproject.org/galaxy`.

Two thoughts:

1. Create a shell wrapper `/srv/galaxy/<instance>/bin/xsede_report` that calls this with `PYTHONPATH` set accordingly (and Galaxy's venv).
2. Create a standalone venv for this script and install the appropriate package(s) in to it.

Reply:
@natefoo Do you have any preference? Seems equal to me.